src/components/CarouselItem.tsx

```diff
@@ -111,7 +111,7 @@ export const CarouselItem = ({ animation, next, prev, swipe, state, index, maxIn
     let animate = 'center';
     if (index === active)
         animate = 'center';
-    else if (index === prevActive)
+    else if (maxIndex>1 && index === prevActive)
     {
         animate = isNext ? 'leftwardExit' : 'rightwardExit';
         if (active === maxIndex && index === 0) animate = 'rightwardExit';
```

Comment:
replace the code in line 117, 118 with the following

```suggestion
    if (maxIndex === 1) {
      if (active === maxIndex && index === 0) animate = 'leftwardExit';
      if (active === 0 && index === maxIndex) animate = 'rightwardExit';
    } else {
      if (active === maxIndex && index === 0) animate = 'rightwardExit';
      if (active === 0 && index === maxIndex) animate = 'leftwardExit';
    }
```